rust/protocol/src/protocol.rs

```diff
@@ -17,6 +20,44 @@ use uuid::Uuid;
 
 pub const CIPHERTEXT_MESSAGE_CURRENT_VERSION: u8 = 3;
 
+/// A [u8] describing the version of the message chain format to use when starting a chain.
+#[derive(Copy, Clone, Eq, PartialEq, Debug, num_enum::TryFromPrimitive, num_enum::IntoPrimitive)]
+#[repr(u8)]
+pub enum MessageVersion {
+    Version2 = 2,
+    /// **\[CURRENT\]**.
+    Version3 = CIPHERTEXT_MESSAGE_CURRENT_VERSION,
+}
+
+impl Default for MessageVersion {
+    fn default() -> Self {
+        Self::Version3
+    }
+}
```

Comment:
This should not have a default, since it's something that ends up in a wire format.

Reply:
Removed!